java/code/src/com/redhat/rhn/frontend/servlets/DumpFilter.java

```diff
@@ -95,53 +95,62 @@ public void init(FilterConfig filterConfig) {
     }
 
     private void logCookies(Cookie[] cookies) {
-        if (cookies == null) {
-            log.debug("There are NO cookies to log");
-            return;
-        }
+        if (log.isDebugEnabled()) {
```

Comment:
why do you need this check if when calling the log you are already calling the debug method? It should check if we are already in debug. 
Some comment for all call of this, but I will comment on it

Reply:
For that particular one I think it should not be needed indeed. This construct is needed when the message requires function calls: those are evaluated before the log level and for instance there is no need to always compute a string representation of a list of systems only for debug mode.

I'll loop over those again to figure out if I have some unneeded ones